server/task/task-distributed/src/main/scala/org/apache/james/task/eventsourcing/cassandra/CassandraTaskExecutionDetailsProjectionDAO.scala

```diff
@@ -26,12 +26,13 @@ import com.datastax.driver.core.{Row, Session}
 import javax.inject.Inject
 import org.apache.james.backends.cassandra.init.{CassandraTypesProvider, CassandraZonedDateTimeModule}
 import org.apache.james.backends.cassandra.utils.CassandraAsyncExecutor
+import org.apache.james.server.task.json.JsonTaskAdditionalInformationsSerializer
 import org.apache.james.task.eventsourcing.cassandra.CassandraTaskExecutionDetailsProjectionTable._
-import org.apache.james.task.{Hostname, TaskExecutionDetails, TaskId, TaskManager, TaskType}
+import org.apache.james.task._
```

Comment:
was there a decision about this?

Reply:
There was a discussion. But no hard decision.
I have just reverted this line for the time being.

Reply:
thank you, I prefer keeping the same rule than on Java for now: no wildcard import.